### PR TITLE
Unify errors rendering (closes #600)

### LINKED ISSUE #600: Unify errors rendering
It's common that runtime errors are wrapped with test run errors. In that case we don't have nice formatting for error messages. We need to unify rendering mechanism somehow.

### PR DESCRIPTION
**Pull request by inikulin**
OK, I've decided to remove `<code>` wrappers in error messages, advantages:
- Solves issues in #600 
- Previously for mediums without color support we've had ambiguous messages, e.g.
`The url argument specifies a URL that uses an unsupported ftp:// protocol.` vs `The "url" argument specifies a URL that uses an unsupported ftp:// protocol.` (current version)
- It just looks better in reports and (surprisingly) more readable due to reduced visual noise.

\cc @helen-dikareva @AlexanderMoskovkin 

So the new rules for error decoration (FYI @DevExpress/testcafe):
- Wrap argument, property, option names with `"`
- For types like `null` or `number` keep string as is.
- Use decoration tags only for URLs or other situations there they can bring additional functionality in some reporters

### CODE REVIEW

**Comment by inikulin** [comment]
Guys, please, review it ASAP because it blocking for #601 and I would like to finish it before Wednesday internal showcase

**Comment by AlexanderMoskovkin** [comment]
I see the first commit in the PR has name `Implement "t.select"`. So after merging all commits will be combined into one commit with this name. Maybe it should have name `Unify errors rendering (closes #600)` ?

File: src/errors/test-run/templates.js [hunk 1]
```diff
@@ -169,25 +169,25 @@ export default {
...
     [TYPE.actionIncorrectKeysError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
+        The "${err.argumentName}" argument contains an incorrect key or key combination.
 
         ${err.getCallsiteMarkup()}
     `),
 
     [TYPE.actionUnsupportedDeviceTypeError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument specifies an unsupported <code>${err.actualValue}</code> device. For a list of supported devices, refer to <a href="http://viewportsizes.com">http://viewportsizes.com</a>
+        The "${err.argumentName}" argument specifies an unsupported "${err.actualValue}" device. For a list of supported devices, refer to <a href="http://viewportsizes.com">http://viewportsizes.com</a>.
```
Comment (helen-dikareva) [inline src/errors/test-run/templates.js]: `...unsupported "${err.actualValue}" device....`
In all other places we don't use "..." for `actualValue`

Comment (inikulin) [inline src/errors/test-run/templates.js]: Because in all other places actual value can't contain spaces

**Comment by inikulin** [comment]
@AlexanderMoskovkin this is because I've started implementation on top of selectors branch, after squash everything will be just fine.

**Comment by inikulin** [comment]
> I see the first commit in the PR has name Implement "t.select". So after merging all commits will be combined into one commit with this name.

Combined commits will be squashed to commit with PR name

File: src/errors/test-run/templates.js [hunk 2]
```diff
@@ -54,61 +54,61 @@ export default {
...
     [TYPE.actionUnsupportedUrlProtocolError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument specifies a URL that uses an unsupported <code>${err.protocol}://</code> protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.
+        The "${err.argumentName}" argument specifies a URL that uses an unsupported ${err.protocol}:// protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.
 
         ${err.getCallsiteMarkup()}
     `),
 
     [TYPE.actionStringArgumentError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument is expected to be a non-empty string, but it was <code>${err.actualValue}</code>.
+        The "${err.argumentName}" argument is expected to be a non-empty string, but it was ${err.actualValue}.
```
Comment (helen-dikareva) [inline src/errors/test-run/templates.js]: `text` in typeAction can contains spaces

File: src/errors/test-run/templates.js (hunk 1 above)
Comment (helen-dikareva) [inline src/errors/test-run/templates.js]: ok, I see

**Comment by inikulin** [comment]
Just bumped version, can be merged without tests

**Comment by helen-dikareva** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
```js
Cannot find the following file(s) to upload:
     ${err.filePaths.map(path => `<code>${escapeHtml(path)}</code>`).join(', ')}
     ${err.getCallsiteMarkup()}
```

Is it this case?
> Use decoration tags only for URLs or other situations there they can bring additional functionality in some reporters

**Comment by inikulin** [comment]
> Is it this case?

Yep

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**